docs/source/cheat_sheet.rst

```diff
@@ -95,9 +95,9 @@ Functions
    x = f  # type: Callable[[int, float], float]
 
    # A generator function that yields ints is secretly just a function that
-   # returns an iterable (see below) of ints, so that's how we annotate it
-   def f(n):
-       # type: (int) -> Iterable[int]
+   # returns an iterator (see below) of ints, so that's how we annotate it
```

Comment:
The "see below" is potentially confusing now since it's referring to a later section that recommends using iterable for "generic iterables (anything usable in 'for')".  I'm open to suggestions as to what to do here.

Reply:
I'd just leave out "(see below)".